scripts/stubtest/allowlist_todo.txt

```diff
@@ -960,20 +937,11 @@ django.core.management.commands.sqlflush.Command.handle
 django.core.management.commands.squashmigrations.Command.handle
 django.core.management.commands.startapp.Command.handle
 django.core.management.commands.startproject.Command.handle
-django.core.management.templates.TemplateCommand.apply_umask
 django.core.management.utils.find_formatters
 django.core.management.utils.run_formatters
 django.core.management.utils.sentinel
-django.core.paginator.Paginator.ELLIPSIS
 django.core.paginator.Paginator.count
 django.core.paginator.Paginator.num_pages
-django.core.paginator.QuerySetPaginator
```

Comment:
Was removed in django 3.1 https://github.com/django/django/blob/stable/3.0.x/django/core/paginator.py#L129-L136